components/modal/modal.js

```diff
@@ -142,15 +145,15 @@ class Modal extends Component {
   }
 
   render() {
-    const { active, fullscreen, children } = this.props;
```

Comment:
The method `componentDidMount` check for that property and call the `open` method.